python/geo2png.py

```diff
@@ -0,0 +1,121 @@
+# trafo -> png
+# Original source https://github.com/vrandezo/wikidata-analytics/blob/6896d639bd124eec7571f1179f27ae2b6bfaa435/geo2png.py
```

Comment:
That is from your additions not from where you copied that code from. So we need @vrandezo to license it.